Silicon/CommonSocPkg/Library/HeciLib/HeciLib.c

```diff
@@ -222,6 +222,49 @@ HeciGetFwCapsSkuMsg (
   return EFI_SUCCESS;
 }
 
+/**
+  Check for Manufacturing Mode
+
+  @param[out]  MeManuMode         Manufacturing mode/production mode.
+
+  @retval EFI_UNSUPPORTED         Current CSME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+
+**/
+EFI_STATUS
+EFIAPI
+HeciGetManufactureMode (

```

Comment:
Currently, HeciLib is common for both APL and CFL. Is this API applicable to APL as well? If not, I'd like to recommend this should be in MeChipsetLib.